### DISCUSSION TITLE
Spectre Importer stuck at job status waiting_to_start

### DISCUSSION THREAD

**Discussion by JamesTeague** [firefly-iii/firefly-iii#4858] [Firefly III]
This issue has been opened before with #3540. It is actually the same issue. I am using version 2.1.2 of the importer and am unable to get the job to start. 

![Screen Shot 2021-06-16 at 13 21 48](https://user-images.githubusercontent.com/6454385/122272471-355b4900-cea6-11eb-9934-ee244e1e3656.png)

The previous issue was a version change; however, being on the latest release I am just looking for a little guidance.

**Comment by JC5** [comment]
The CSRF token issue seems to point to a config error. I see http + https used in a mix, is that correct?

**Reply by JamesTeague** [reply]
That seems to be true for the endpoint to the importer (https) and the configured endpoint to firefly (http). I can certainly change them both to use https. However in that screenshot the mix seems to be both for the importer endpoint. Which I am unsure why http would be used. Is there a place where I may have misconfigured it to use http?

**Reply by JamesTeague** [reply]
Although the http call there on the screenshot is for the stylesheets, all other requests in the network tab are all using `https`.

**Comment by JC5** [comment]
Alright a CSRF token mismatch means that something in the reverse proxy (I suspect?) configuration is wrong. Are you using a reverse proxy for the Spectre importer, by any chance?

**Reply by JamesTeague** [reply]
Yes. I have the importer running in a container in a kubernetes cluster. And I'm using an NGINX ingress to connect to it via the browser. So it is going through a proxy.

**Reply by JC5** [reply]
Mmm, sorry, I was a little late to the party. Do you have TRUSTED_PROXIES set to **?

**Reply by JamesTeague** [reply]
No worries. Not getting notifications from GitHub either.

---

Is this something I should pass as an argument to the container? Because if so, I have not and can't try that.

**Reply by JC5** [reply]
Mmm, that was my idea but otherwise I'm not sure. Does your reverse proxy pass on stuff like the http host?

**Reply by JamesTeague** [reply]
I'm not sure I understand the question. The reverse proxy is just NGINX. If the TRUSTED_PROXIES flag is an environment variable I should pass to the docker container for the importer I can try that if that's what you are suggesting. That's a quick thing to test.
